src/JsonSchema/SchemaFactory.php

```diff
@@ -327,6 +327,9 @@ private function getFactoryOptions(array $serializerContext, array $validationGr
     {
         $options = [];
 
+        /* @see https://github.com/symfony/symfony/blob/v5.1.0/src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php */
+        $options['enable_getter_setter_extraction'] = true;
```

Comment:
IMO you could add it directly above :slightly_smiling_face:

Reply:
yeah I thought about this as well.
But IMHO it won't look good in case when it will be needed to add one more key-value there with explanation comment.

Reply:
Why not like this:
```php
$options[
    /* @see https://github.com/symfony/symfony/blob/v5.1.0/src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php */
    'enable_getter_setter_extraction' => true,
    'another_option' => 'foo',
];
```

Reply:
I'm okay to do this if you still think that it is critical. Should I?

Reply:
It's not critical at all. It's only some code style :slightly_smiling_face:

Reply:
done in this case :+1: